### PR TITLE
fix: V3 Switch visual update

### PR DESCRIPTION
**Pull request by burakukula**
Hey!

Visual alignment of the Switch component, not a perfect solution in my opinion. I need to overwrite icons sizes in order to make it work like required. This should be fixed in v4 when we will deal with icons - we would have 12px size icon. I'm also adding the size prop.

As an alternative, we could use both icons in the background for `:before` and `:after` elements, but we the icons could run out of date. 🤷‍♀️ 

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Switch/Switch.css [hunk 1]
```diff
@@ -1,26 +1,67 @@
...
+  }
+}
+
+.Switch--small {
+  width: calc(1rem * (28 / var(--font-base-default)));
+  height: calc(1rem * (12 / var(--font-base-default)));
+
+  & + .Switch__label {
+    & .Switch__icon--done {
+      top: 3px;
```
Comment (burakukula) [inline packages/forma-36-react-components/src/components/Switch/Switch.css:42]: Not pretty, unfortunately.

Comment (denkristoffer) [reply packages/forma-36-react-components/src/components/Switch/Switch.css:42]: 🤯

**Review by denkristoffer — APPROVED** [review]

**Comment by suevalov** [comment]
@burakukula looks like we need to add `pointer-events: none` to icons in the component. `close` icon steals hover cursor from the container and `cursor: not-allowed` is fickering.

otherwise - looks beatiful!

**Review by suevalov — APPROVED** [review]

File: packages/forma-36-react-components/src/components/Switch/Switch.css (hunk 1 above)
Comment (mshaaban0) [reply packages/forma-36-react-components/src/components/Switch/Switch.css:42]: for icon alignment?

**Review by mshaaban0 — APPROVED** [review]

**Review by fabe — APPROVED** [review]

**Pull request merged**